smaps/src/main/java/com/google/smaps/Homepage.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.smaps;
+
+import com.google.appengine.api.utils.SystemProperty;
+import java.io.IOException;
+import java.util.Properties;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(name = "Homepage", value = "/home")
+public class Homepage extends HttpServlet {
```

Comment:
As per the style guide here- https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required
A javadoc is needed for the public class.